src/components/screens/DocsScreen/VersionSelector.tsx

```diff
@@ -1,113 +1,59 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import { styled } from '@storybook/theming';
-import { Link, Subheading, TooltipLinkList, styles, WithTooltip } from '@storybook/design-system';
+import { Menu } from '@storybook/components-marketing';
 
 import GatsbyLinkWrapper from '../../basics/GatsbyLinkWrapper';
 import buildPathWithFramework from '../../../util/build-path-with-framework';
 
-const { breakpoint, color, typography } = styles;
+const stylizeVersion = ({ label, string }: { label?: string; string: string }) =>
+  label ? `${string} (${label})` : string;
 
-const Wrapper = styled.div`
-  font-size: ${typography.size.s2}px;
-`;
-
-const Version = styled.span`
-  color: ${color.darker};
-  display: none;
-  @media (min-width: ${breakpoint * 1}px) {
-    display: inline;
-  }
-`;
-
-const VersionLink = styled(Link)`
-  svg {
-    transform: rotate(90deg);
-  }
-`;
+interface Stable {
+  version: number;
+  string: string;
+  label?: 'latest';
+}
 
-const LinkHeading = styled(Subheading)`
-  display: block;
-  font-size: 10px;
-  font-weight: 800;
-  letter-spacing: 4.29px;
-  line-height: 18px;
-  color: ${color.mediumdark};
-  padding: 7px 15px;
-  border-bottom: 1px solid ${color.border};
-  ${(props) => props.withTopBorder && `border-top: 1px solid ${color.border};`}
-`;
+interface PreRelease {
+  version: number;
+  string: string;
+  label: 'alpha' | 'beta' | 'rc';
```

Comment:
`7.0` also had a `'future'` label initially, but I'm not sure if that was just a one-time thing. Probably can't hurt to include it.